analytics/api/src/main/java/mil/nga/giat/geowave/analytic/clustering/NeighborData.java

```diff
@@ -102,6 +102,7 @@ public int compareTo(
 
 	@Override
 	public String toString() {
-		return (id == null ? "" : Hex.encodeHexString(id.getBytes()) + ":") + element.toString() + "(" + this.distance + ")";
+		return (id == null ? "" : Hex.encodeHexString(id.getBytes()) + ":") + element.toString() + "(" + this.distance
+				+ ")";
```

Comment:
The formatter defines the following, which is why this one occurs:
`<setting id="org.eclipse.jdt.core.formatter.lineSplit" value="120"/>`